ngraph/frontend/onnx_import/include/onnx_import/core/value_info.hpp

```diff
@@ -75,7 +75,7 @@ namespace ngraph
 
             const std::string& get_name() const { return m_value_info_proto->name(); }
             const PartialShape& get_shape() const { return m_partial_shape; }
-            const element::Type& get_element_type() const
+            const element::Type get_element_type() const
```

Comment:
When returning by value it's recommeded to drop the const.

```suggestion
            element::Type get_element_type() const
```

Reply:
Fixed